csi/util.go

```diff
@@ -218,6 +218,10 @@ func getVolumeOptions(volOptions map[string]string) (*longhornclient.Volume, err
 		vol.NodeSelector = strings.Split(nodeSelector, ",")
 	}
 
+	vol.BackendStoreDriver = string(longhorn.BackendStoreDriverTypeLonghorn)
+	if driver, ok := volOptions["backendStoreDriver"]; ok {
```

Comment:
Let's have some example manifests for SPDK in https://github.com/longhorn/longhorn/tree/7b3b230f471071f08b9621a165fab7887e087491/examples.